multi_press/automation/jsr223/python/community/multi_press/multi_press.py

```diff
@@ -0,0 +1,178 @@
+"""
+Copyright September 23, 2020 Sascha Volkenandt
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import traceback
+import core
+from core.osgi import register_service, unregister_service
+from core.log import logging, LOG_PREFIX
+from java.util.concurrent import TimeUnit
+from org.eclipse.smarthome.core.library.types import StringType
+from org.eclipse.smarthome.core.thing.profiles import ProfileTypeUID, ProfileFactory, \
```

Comment:
I actually doubt that these are in `scope`. If they are not, maybe another way to handle the move to OH 3 is to put try/except around the import and if the import from org.eclipse.smarthome fails import from org.openhab instead.